packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -168,7 +169,7 @@ class Auth {
 
     constructor(config: AuthConfig) {
         this.client = new ShopperLogin({
-            proxy: config.proxy,
+            proxy: config.slasProxy,
```

Comment:
Using the regular config.proxy would send calls to `/mobify/proxy/api/` which does not include the logic for inserting the client secret at this time